components/statuslight/metadata/statuslight.yml

```diff
@@ -6,8 +6,23 @@ sections:
       This component can be modified via its `--mod-*` prefixed custom properties. A list of those prefixed custom properties can be found <a class="spectrum-Link" href="https://github.com/adobe/spectrum-css/tree/main/components/statuslight/metadata/mods.md">here</a>.
   - name: Migration Guide
     description: |
-      ### T-shirt sizing
-      Status Light now supports t-shirt sizing and requires that you specify the size by adding a `.spectrum-StatusLight--size*` class.
+      ### Version x.x.x
```

Comment:
I haven't run the changeset yet to get the version number.

Reply:
@rise-erpelding ran the changeset: https://github.com/adobe/spectrum-css/pull/2818/commits/90e39119c755e1d32874e1ef38b4f7dab24f75f2